app/controllers/api/calls_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::CallsController < ApplicationController
   skip_before_action :verify_authenticity_token, raise: false
 
   def create
-    success = verify_recaptcha(action: params[:recaptcha_action], minimum_score: 0.5)
+    success = verify_recaptcha(secret_key: Settings.recaptcha2.secret_key)
```

Comment:
@osahyoun - We're gonna have (small) conflicts since I restructured #create a little bit. Should we combine our branches and do a release to get it all live at the same time? Let me know what you think when you come online today.

Reply:
OK - there's only a pending comment on some error copy, otherwise it's good to go. We can even add that later. I'll merge - I'd really like to get this live for Nick and Nabil. Nick says that his action only really makes sense this week so if only possible, would be really great to get this live today so he could use the calling tool.

Reply:
👍